plugin/storage/grpc/config/config.go

```diff
@@ -55,17 +84,21 @@ type PluginBuilder interface {
 }
 
 // Build instantiates a PluginServices
-func (c *Configuration) Build(logger *zap.Logger, tracerProvider trace.TracerProvider) (*ClientPluginServices, error) {
-	return c.buildRemote(logger, tracerProvider, grpc.NewClient)
+func (c *ConfigV2) Build(logger *zap.Logger, tracerProvider trace.TracerProvider) (*ClientPluginServices, error) {
```

Comment:
I suggest 
```
func NewRemoteStorage(*ConfigV2)
```

leave Build where it is, with Configuration, but re-implement as
```
func (c *Configuration) Build(...) {
    v2cfg = c.AsV2()
    s := NewRemoteStorage(v2cfg)
}
```

Reply:
PTAL, is this what you intended?